README.md

```diff
@@ -0,0 +1,169 @@
+# Air Pollution metric
+
+Mobile version of SPA that indicates the levels of pollution in the air in the countries of Europe (more continents will be added eventually) - React-Redux was used for the creation and ran Test using Jest.
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3 id="title"><b>SPA website</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Air Pollution metric <a name="about-project"></a>
+
+Discover the current levels of pollution in the air with this mobile website, you can see a graph indicating the distribution of the elements present in the region!
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JS
+- React
+- Redux
+- Jest
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li> 
+    <li><a href="https://create-react-app.dev/docs/getting-started">React-app</a></li> 
+    <li><a href="https://www.w3schools.com/js/">Javascript</a></li> 
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+1 Fetch API for pollution data<br>
+2 Graph for pollution values<br>
+3 Search bar<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"> here is Love Demo</a>
+
+- [Live Demo](https://air-pollution-app.onrender.com)
+- Project Loom video explaination [here](https://www.loom.com/share/34264643b5a342cf85d86c689939ab2a)
```

Comment:
- [ Required ] I think it would be nice if you record a video presentation and a link to the video on your readme as requested by the previous reviewer. Currently, when I click on this link it displays the message shown in the screenshot below which is not professional. The content of the video should include:
   
   - A description of the project.
   - A demo of the project features.
   - You should also highlight some interesting piece of code or something you built that you are very proud of.

## Error when video link is clicked on
![Screenshot from 2023-09-17 00-26-53](https://github.com/kazim110/AirPollutionApp/assets/16392046/0866716b-2da3-4996-be7e-86bc6b401771)


## Project requirements
![Screenshot from 2023-09-17 00-24-25](https://github.com/kazim110/AirPollutionApp/assets/16392046/1d123c72-9794-44e1-bc15-27e0c3b24bd7)